x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/configured_sources_list.tsx

```diff
@@ -70,7 +70,7 @@ export const ConfiguredSourcesList: React.FC<ConfiguredSourcesProps> = ({
         <React.Fragment key={i}>
           <EuiFlexItem>
             <EuiPanel paddingSize="s">
-              <EuiFlexGroup alignItems="center" gutterSize="none" responsive={false}>
+              <EuiFlexGroup alignItems="center" gutterSize="none" responsive={false} className="eui-fullHeight">
```

Comment:
This is a prettier line width issue it looks like

```suggestion
              <EuiFlexGroup
                alignItems="center"
                gutterSize="none"
                responsive={false}
                className="eui-fullHeight"
              >
```

Reply:
Seems like it, thanks for the save! 👍🏽

Reply:
FWIW, I totally recommend downloading the Prettier plugin for your IDE of choice (Sublime, VSCode, and Atom should all have it) and setting it auto-fix formatting issues on save. It's super great and saves me a ton of time on the daily!

Reply:
Good tip! Scotty also just mentioned that in our 1:1 and I've now got it install on Atom.